server/api/crud/model_monitoring/deployment.py

```diff
@@ -395,30 +281,21 @@ def _initial_model_monitoring_stream_processing_function(
 
         return function
 
-    def _get_model_monitoring_batch_function(
-        self, image: str, function_name: str = "model-monitoring-batch"
-    ):
+    def _get_model_monitoring_batch_function(self, image: str, function_name: str):
```

Comment:
please rename to so some think that related to the controller, and also the docs because this function not initialize the monitoring batch.

Also In this class please remove   - "2. model monitoring batch" from the docs of the init

Reply:
Yep, I thought about changing the name but didn't. I will do